jobs/build-node-image.Jenkinsfile

```diff
@@ -68,6 +68,20 @@ lock(resource: "build-node-image") {
         def timestamp = now.format(java.time.format.DateTimeFormatter.ofPattern("yyyyMMddHHmm"))
         def (registry_staging_repo, registry_staging_tags, registry_prod_repo, registry_prod_tags) = pipeutils.get_ocp_node_registry_repo(pipecfg, params.RELEASE, timestamp)
 
+        // Get the tag that's unique
+        def unique_tag = ""
+        for (tag in registry_prod_tags) {
+            if (tag.contains(timestamp)) {
+                if (unique_tag != "") {
+                    error("Multiple unique tags found: ${registry_prod_tags}")
+                }
+                unique_tag = tag
+            }
+        }
+        if (unique_tag == "") {
+            error("No unique tag found: ${registry_prod_tags}")
+        }
```

Comment:
hmm. I think we configured the devel RHCOS pipeline to not push a unique tag (i.e. we actually wanted GC to clean up).

If that's true this will fail in the devel RHCOS pipeline.

Reply:
In this case the unique tag, is the one using the timestamp, so it won't be GC

Reply:
I think you are missing my point.

In the configuration for the devel pipeline there is no unique (unique to this `build-node-image` run) label.

So the result after we merge this PR would be:

```
ERROR: No unique tag found: 4.19-node-image-devel
```

Reply:
Fixed, thanks for catching!